### PR TITLE
feat: bump @noble/hashes and use bip39 deps instead

### PR DESCRIPTION
**Pull request by davidfragalaureano**
Bumps:

- `@noble/hashes` `1.1.3 ` --> `1.3.2`

Replaces:

- `@scure/bip39@1.2.1` --> `bip39@3.0.2`

Adds `files` entry to `package.json`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,5 +64,5 @@
     "typedoc": "^0.23.20",
     "typescript": "4.8.2"
   },
-  "version": "1.19.4"
+  "version": "1.19.4-exodus.0"
```
Comment (davidfragalaureano) [inline package.json:59]: I'll release the package with this version in npm once the PR gets merged.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,17 @@
...
+    "src/*",
+    "dist/*",
+    ".eslintignore",
+    ".eslintrc.js",
+    ".nvmrc",
+    ".prettierignore",
+    ".versionrc.json",
+    "jest.config.js",
+    "!src/tests"
+  ],
```
Comment (633kh4ck) [inline package.json]: Why not just `dist`? Do we need the rest of it?

Comment (davidfragalaureano) [reply package.json]: I don't think so, `dist/` will be enough: [8f5cf2f](https://github.com/ExodusMovement/aptos-sdk-fork/pull/1/commits/8f5cf2f2b8cc3e1255dfa6844877445e0c78a614)

**Review by 633kh4ck — APPROVED** [review]
tACK (tests pass)

**Pull request merged**